aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -513,12 +309,8 @@ The handler settings can also be provided via environment variables:
 > [!WARNING]
 > Do **not** leave debug logging enabled in the deployment for longer than required to troubleshoot an issue. The size of the log isn't limited. Leaving the debug log enabled can exhaust the available disk space and crash the server or app service.
 
-::: moniker-end
-
 See [Configuration with web.config](#configuration-with-webconfig) for an example of the `aspNetCore` element in the *web.config* file.
 
-::: moniker range=">= aspnetcore-3.0"
-
 ## Modify the stack size
```

Comment:
It might be worth pointing out this only applies to the InProcess hosting model.